### PR TITLE
fix(react-components): allow mixed case rule metadata key

### PR DESCRIPTION
**Pull request by haakonflatval-cognite**
#### Type of change

Bug 

#### Jira ticket :blue_book:

https://cognitedata.atlassian.net/browse/

## Description :pencil:

A recent change removed the support for mixed-case metadata keys in rule-based color expressions. This re-adds support.

## How has this been tested? :mag:

Linking into Search/fusion - older rules now work again

## Checklist :ballot_box_with_check:

- [x] I am proud of this feature.
- [ ] I have performed a self-review of my own code.
- [x] I have added PR type (Feat, Bug, Chore, Test, Docs, Style, Refactor) to the PR title.
- [ ] I have manually tested this for different scenarios (different models, formats, devices, browsers).
- [ ] I have commented my code in hard-to-understand areas.
- [ ] I have made corresponding changes to the public documentation.
- [ ] I have added unit and visuals tests to prove that my feature works to the best of my ability.
- [ ] I have refactored the code for readability to the best of my ability.
- [ ] I have checked that my changes do not introduce regressions in the public documentation.
- [ ] I have outlined any known defects / lacking capabilities in the contents of this PR.
- [ ] I have listed any remaining work that should follow this PR in the description or jira/miro/etc.
- [ ] I have added TSDoc to any public facing changes.
- [ ] I have added "developer documentation" in any package README.md that is related / applicable to this PR.
- [ ] I have noted down and am currently tracking any technical debt introduced in this PR.
- [ ] I have thoroughly thought about the architecture of this implementation.
- [ ] I have asked peers to test this feature.

### CODE REVIEW

File: react-components/src/components/ViewerAnchor/ViewerAnchor.tsx [hunk 1]
```diff
@@ -190,7 +190,7 @@ function computeCombinedTranslationCss(
   const inherentTranslation = `translateX(calc(${pointPositionOnScreen.x}px + ${stickyCompensation.x}px))
 translateY(calc(${pointPositionOnScreen.y}px + ${stickyCompensation.y}px))`;
 
-  const style = { ...inputStyle } ?? {};
+  const style = { ...inputStyle };
```
Comment (danpriori) [inline react-components/src/components/ViewerAnchor/ViewerAnchor.tsx:193]: Is this related to this PR? I didnt understand the purpose of this change 🤔

Comment (haakonflatval-cognite) [reply react-components/src/components/ViewerAnchor/ViewerAnchor.tsx:193]: No :see_no_evil:
